### ISSUE TITLE
Application submission request fails with 404 

### ISSUE THREAD

**Issue by pbn4** [bloom-housing/bloom#644]
Request metadata:
```
Request URL: http://localhost:3100/listings/applications
Request Method: POST
Status Code: 404 Not Found
Remote Address: [::1]:3100
Referrer Policy: no-referrer-when-downgrade
```

@bk3c I guess changes to base url caused frontend to consider `http://localhost:3100/listings` as base instead of `http://localhost:3100`.

**Comment by bk3c** [comment]
I think this should be fixed in #636. You definitely need to change your local .env, since that's not in source control, but I think the template files are properly updated.

**Comment by kathyccheng** [comment]
Closed in #636
